src/app/server/Server.h

```diff
@@ -135,6 +138,12 @@ struct ServerInitParams
     // Operational certificate store with access to the operational certs in persisted storage:
     // must not be null at timne of Server::Init().
     Credentials::OperationalCertificateStore * opCertStore = nullptr;
+
+#ifdef CHIP_DEVICE_CONFIG_ICD_SERVER_ENABLE
+    // ICD Manager: Must be injected. Used to process logic for ICD server behaviours
+    // Must NOT be initialized before provided
```

Comment:
I don't understand what this "Must NOT" comment is trying to say.

Reply:
this was removed